### ISSUE TITLE
preinstall script failing

### ISSUE THREAD

**Issue by cosigyn** [blomqma/next-rest-framework#93]
```bash
PS C:\Users\REDACTED_USER\Documents\Clanware\clanware-api> npm install --save next-rest-framework
npm ERR! code 1
npm ERR! path C:\Users\REDACTED_USER\Documents\Clanware\clanware-api\node_modules\next-rest-framework
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c mkdir -p dist && touch ./dist/cli.js && echo "#!/usr/bin/env node" > ./dist/cli.js
npm ERR! A subdirectory or file dist already exists.
npm ERR! Error occurred while processing: dist.

npm ERR! A complete log of this run can be found in: C:\Users\REDACTED_USER\AppData\Local\npm-cache\_logs\2023-11-04T14_05_42_556Z-debug-0.log
```

Full log:
[2023-11-04T14_05_42_556Z-debug-0.log](https://github.com/blomqma/next-rest-framework/files/13256833/2023-11-04T14_05_42_556Z-debug-0.log)

**Comment by cosigyn** [comment]
Works in WSL with Ubuntu, but not on windows.

**Comment by blomqma** [comment]
Thanks, the preinstall step is no longer needed so removing the UNIX-specific commands should address this issue. This fix has been now introduced in `v3.4.2`.

**Comment by cosigyn** [comment]
Thansk!
